### PR TITLE
Add ERC: Transfer With Authorization

### PR DESCRIPTION
**Pull request by dongri**
Ref: https://github.com/ethereum/ERCs/pull/504

### CODE REVIEW

**Review by Allzz77123 — APPROVED** [review]

File: ERCS/erc-transfer-with-authorization.md [hunk 1]
```diff
@@ -0,0 +1,536 @@
+---
+eip: 3009
```
Comment (abcoathup) [inline ERCS/erc-transfer-with-authorization.md:2]:
```suggestion
eip: 7758
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.

**Comment by abcoathup** [comment]
Please close https://github.com/ethereum/EIPs/pull/8819

File: ERCS/erc-transfer-with-authorization.md (hunk 1 above)
Comment (dongri) [reply ERCS/erc-transfer-with-authorization.md]: @abcoathup Thank you for the review
https://github.com/ethereum/ERCs/pull/598/commits/ea63268cda2ca0a21eb7d7eec173296fc9f2ed66
* Update eip number to 7758
* Change filename

File: ERCS/erc-7758.md [hunk 2]
```diff
@@ -0,0 +1,530 @@
+---
+eip: 7758
+title: Transfer With Authorization
+author: Peter Jihoon Kim (@petejkim), Kevin Britz (@kbrizzle), David Knott (@DavidLKnott), Dongri Jin (@dongri)
+discussions-to: https://github.com/ethereum/EIPs/issues/3010
```
Comment (abcoathup) [inline ERCS/erc-7758.md:5]: Please create a discussions topic in Eth Magicians.  It just needs to contain a link to this PR.
https://ethereum-magicians.org/

Comment (dongri) [reply ERCS/erc-7758.md:5]: I signed up, but it seems I can't post yet. I'll add the link once I can post.

Comment (dongri) [reply ERCS/erc-7758.md]: https://github.com/ethereum/ERCs/pull/598/commits/164054885890db83a32b1659178e0910d64a0ce3
@abcoathup DONE!

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**